### DISCUSSION TITLE
How to add LDAP

### DISCUSSION THREAD

**Discussion by renatochaves1999** [uvdesk/community-skeleton#707] [Q&A] [answered]
Good afternoon,
Is there any way to add users in LDAP in uvdesk

**Comment by komal-sh-27** [comment]
@renatochaves1999 

We are informing to you, at this moment this LDAP feature is not available in uvdesk.

Kind Respects,
Uvdesk Team

**Reply by renatochaves1999** [reply]
Good morning,

Do you have any predictions for when this feature will be available?

**Comment by komal-sh-27** [comment, marked as answer]
@renatochaves1999 

As of now, our UVdesk helpdesk does not support **LDAP integration** in either of the versions - **UVdesk helpdesk Saas (Subscription Based) or Open Source (Free)**. We cannot share a timeframe to include this feature by default as our technical team is working with limited capacity currently.

However, we can provide this integration in the UVdesk helpdesk ticketing system through custom development (a paid service) from our end.

If you are interested in proceeding further then kindly contact our sales team through our [ticketing portal](https://support.uvdesk.com/en/customer/create-ticket/) so that we can share with you the customization cost.

Thanks and Regards,
Uvdesk Team
